docs/operation_center/rules_catalog.md

```diff
@@ -0,0 +1,233 @@
+# Detection Rules Management
+
+## Rules Catalog:
+
+### Security profile MITRE ATT&CK
+
+The MITRE ATT&CK framework is a comprehensive matrix of tactics and techniques used by threat actors and defenders to better classify attacks and assess an organization's risk.
+Every time you enable a rule, it appears on the Framework in blue in one or many cells.
+Each cell represents an attack technique. The cells are clickable and enable you to see or disable the rules activated in each one.
+You can see how many rules are enabled in a cell by hovering over it.
+The color changes depending on the number of rules activated in one cell. The blue gets darker when more rules are enabled and a white cell means that no rule is activated in it.
+
+### Search bar
```

Comment:
information concerning search bar and filters should be grouped in a single part 'filtering the rules list'. Search bar, filters in the left, effort level, verified and selection of a case in the MITRE ATT&CK matrix are all cumulative for rule filtering.